### PR TITLE
Add an option for RPC Address

### PR DESCRIPTION
**Pull request by skonhwang**

### CODE REVIEW

File: plugins/node_peer_count/main.go [hunk 1]
```diff
@@ -26,12 +27,14 @@ var (
...
 	port    int
 	minPeer int
+	tenderPort uint
 )
 
 func init() {
 	flag.StringVar(&addr, "addr", defaultAddr, "IP Address(e.g. 0.0.0.0, 127.0.0.1)")
 	flag.IntVar(&port, "port", defaultPort, "Port number, default 9091")
 	flag.IntVar(&minPeer, "minPeer", defaultPeer, "minimum peer count, default 5")
+	flag.UintVar(&tenderPort, "tendermintPort", defaultPeer, "tendermint port, default 26657")
```
Comment (gnongs) [inline plugins/node_peer_count/main.go:37]: I think `defaultPeer` have to change another variable.

Comment (skonhwang) [reply plugins/node_peer_count/main.go]: Thanks, I modified.

**Review by gnongs — APPROVED** [review]
LGTM!

File: rpc/cosmos/net_info.go [hunk 2]
```diff
@@ -80,8 +80,9 @@ type netInfo struct {
 	} `json:"result"`
 }
 
-func GetNpeers() (string, error) {
-	req, err := http.NewRequest(http.MethodGet, "http://localhost:26657/net_info", nil)
+func GetNpeers(tenderPort uint) (string, error) {
+	url := fmt.Sprintf("http://localhost:%d/net_info", tenderPort)
```
Comment (heejin-github) [inline rpc/cosmos/net_info.go:84]: I think we need to consider to get the data from another rpc endpoint, not only localhost. 
please refer to the another plugins.

- https://github.com/dsrvlabs/vatz-plugin-cosmoshub/blob/main/rpc/cosmos/api.go#L55
- https://github.com/dsrvlabs/vatz-plugin-cosmoshub/blob/main/rpc/cosmos/health.go#L12

**Review by heejin-github — CHANGES_REQUESTED** [review]

File: rpc/cosmos/net_info.go (hunk 2 above)
Comment (heejin-github) [reply rpc/cosmos/net_info.go:84]: governance alarm plugin can also get the data from localhost only.

- https://github.com/dsrvlabs/vatz-plugin-cosmoshub/blob/main/rpc/cosmos/governance.go#L58

Comment (skonhwang) [reply rpc/cosmos/net_info.go]: @heejin-github , I added an option for rpc address and removed tendermint port.
Thanks!

**Review by heejin-github — APPROVED** [review]
LGTM

**Pull request closed without merging**